tests/test_smoke.py

```diff
@@ -40,6 +40,8 @@ def run(cmd, check=True):
 @contextmanager
 def chdir(folder):
```

Comment:
I need to remove this completely (will do on a separate PR) as it is no longer used